spec/Cabin/Infotainment.vspec

```diff
@@ -267,4 +267,27 @@ SmartphoneProjection.SupportedMode:
   allowed: [ 'ANDROID_AUTO', 'APPLE_CARPLAY', 'MIRROR_LINK', 'OTHER' ]
   description: Supportable list for projection.
 
+SmartphoneScreenMirroring:
```

Comment:
In https://github.com/COVESA/vehicle_signal_specification/blob/master/spec/Cabin/Infotainment.vspec#L247 we have 

```
SmartphoneProjection:
  type: branch
  description: All smartphone projection actions.
```

I think it would be good to add some more information on what is the difference between `SmartphoneProjection` and `SmartphoneScreenMirroring`.

My assumption (for now)

- `SmartphoneProjection` - Expose or control specific applications on the Smartphone on the vehicle infotainment system
- `SmartphoneScreenMirroring` - Mirror the whole screen of the Smartphone on the vehicle infotainment system

Or is there a better definition?

Reply:
Thanks erik. I think your definitions are good for me. 
I made comments for each of the function.